Fix build, failed junit due to typo in unit test

**Describe the issue**: please provide a clear and concise description of what the bug is.  

Failure in ContestSnakeYAMLLoader, typo in junit, uses sumitMTC not SumitMTC

    [junit] Caused by: java.io.FileNotFoundException: samps/contests/SumitMTC/config/contest.yaml


**To Reproduce**: please describe, in order, the steps to reproduce the behavior:

Merge/push, if not unit errors, bug fixed.

**Expected behavior**: please provide a description of what you *expected* to happen when you took the above steps:  

clean/no errors in build job on gitLab.

**Actual behavior**: please describe what _actually_ happened when you took the above steps:


**Environment**: please complete the following information:

Unix

**Log Info**:  PC^2 generates logs (in the **logs** folder for client modules; under the **profiles** folder for the PC2 Server).  Please review the relevant logs and provide copies of them as attachments -- particularly if the logs show Exceptions being thrown.  

**Screenshots**:  if applicable, please add attachments showing screenshots to help explain the issue.

**Additional context**: please add any other context about the issue here.

--- a/test/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoaderTest.java
+++ b/test/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoaderTest.java
@@ -2901,13 +2901,13 @@ public class ContestSnakeYAMLLoaderTest extends AbstractTestCase {
     }
     
     /**
-     * Test load of input format validator using SumitMTC sample.
+     * Test load of input format validator using sumitMTC sample.
      * 
      * @throws Exception
      */
     public void testLoadInputFormatValidator() throws Exception {
 
-        IInternalContest con = loadSampleContest(null, "SumitMTC");
+        IInternalContest con = loadSampleContest(null, "sumitMTC");
         assertNotNull(con);
 
         // expecting input format validator defined
@@ -2917,7 +2917,7 @@ public class ContestSnakeYAMLLoaderTest extends AbstractTestCase {
         
         String valiatorFileName = "valid.bat";
 
-        String ifvfilename = "samps/contests/SumitMTC/config/sumit/input_format_validators/"+valiatorFileName;
+        String ifvfilename = "samps/contests/sumitMTC/config/sumit/input_format_validators/"+valiatorFileName;
         assertFileExists(ifvfilename);
 
         Problem p = problems[0];
@@ -3139,11 +3139,11 @@ public class ContestSnakeYAMLLoaderTest extends AbstractTestCase {
      */
     public void testLanguageLoadJudgeCmdLine() throws Exception {
         
-//        String configDir = getTestSampleContestDirectory( "SumitMTC") + File.separator + IContestLoader.CONFIG_DIRNAME;
+//        String configDir = getTestSampleContestDirectory( "sumitMTC") + File.separator + IContestLoader.CONFIG_DIRNAME;
 //        String yamlFile = configDir + File.separator + IContestLoader.DEFAULT_CONTEST_YAML_FILENAME;
 //        editFile(yamlFile);
 
-        IInternalContest contest = loadSampleContest(null, "SumitMTC");
+        IInternalContest contest = loadSampleContest(null, "sumitMTC");
         assertNotNull(contest);
         
         Language[] languages = contest.getLanguages();

--- a/test/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoaderTest.java
+++ b/test/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoaderTest.java
@@ -2901,13 +2901,13 @@ public class ContestSnakeYAMLLoaderTest extends AbstractTestCase {
     }
     
     /**
-     * Test load of input format validator using sumitMTC sample.
+     * Test load of input format validator using SumitMTC sample.
      * 
      * @throws Exception
      */
     public void testLoadInputFormatValidator() throws Exception {
 
-        IInternalContest con = loadSampleContest(null, "sumitMTC");
+        IInternalContest con = loadSampleContest(null, "SumitMTC");
         assertNotNull(con);
 
         // expecting input format validator defined
@@ -2917,7 +2917,7 @@ public class ContestSnakeYAMLLoaderTest extends AbstractTestCase {
         
         String valiatorFileName = "valid.bat";
 
-        String ifvfilename = "samps/contests/sumitMTC/config/sumit/input_format_validators/"+valiatorFileName;
+        String ifvfilename = "samps/contests/SumitMTC/config/sumit/input_format_validators/"+valiatorFileName;
         assertFileExists(ifvfilename);
 
         Problem p = problems[0];
@@ -3139,17 +3139,16 @@ public class ContestSnakeYAMLLoaderTest extends AbstractTestCase {
      */
     public void testLanguageLoadJudgeCmdLine() throws Exception {
         
-//        String configDir = getTestSampleContestDirectory( "sumitMTC") + File.separator + IContestLoader.CONFIG_DIRNAME;
+//        String configDir = getTestSampleContestDirectory( "SumitMTC") + File.separator + IContestLoader.CONFIG_DIRNAME;
 //        String yamlFile = configDir + File.separator + IContestLoader.DEFAULT_CONTEST_YAML_FILENAME;
 //        editFile(yamlFile);
 
-        IInternalContest contest = loadSampleContest(null, "sumitMTC");
+        IInternalContest contest = loadSampleContest(null, "SumitMTC");
         assertNotNull(contest);
         
         Language[] languages = contest.getLanguages();
         
         assertEquals("Expecting language count ", 7, languages.length);
-        
         
         for (Language language : languages) {
             if ("Perl".equals(language.getDisplayName())){

